### PR TITLE
"mkdir -p /etc/cni/net.d" requires sudo

### PR DESCRIPTION
**Pull request by nc-ruth**

### CODE REVIEW

**Comment by haircommander** [comment]
/ok-to-test
bot, test this pull request
hey @nc-ruth thanks for the PR, can you sign it off with `git commit --amend -s` and `git push -f origin patch-1`

**Comment by nc-ruth** [comment]
Hi @haircommander,
I have the following issue, when trying to do "git push -f origin patch-1"

````
ruth@PF1371KE:~/tmp/libpod$ git push -f origin patch-1
error: src refspec patch-1 does not match any.
error: failed to push some refs to 'git@github.com:containers/libpod.git'
```

**Comment by haircommander** [comment]
what's the output of `git branch` and `git remote -v` 
ultimately, you need to force push the changes to this branch (I think it's patch-1) in your personal repo (nc-ruth/libpod)

File: install.md [hunk 1]
```diff
@@ -252,7 +252,7 @@ A proper description of setting up CNI networking is given in the [`cni` README]
 Using the CNI plugins from above, a more basic network config is achieved with:
 
 ```bash
-mkdir -p /etc/cni/net.d
+sudo mkdir -p /etc/cni/net.d
 curl -qsSL https://raw.githubusercontent.com/containers/libpod/master/cni/87-podman-bridge.conflist | sudo tee /etc/cni/net.d/99-loopback.conf
```
Comment (rhatdan) [inline install.md]: The next line would require sudo also, since it is attempting to write to /etc/cni/net.d

Comment (nc-ruth) [reply install.md]: @rhatdan the "sudo tee" handles that :)

**Comment by nc-ruth** [comment]
> what's the output of `git branch` and `git remote -v`
> ultimately, you need to force push the changes to this branch (I think it's patch-1) in your personal repo (nc-ruth/libpod)

Thank you :) - just pushed it ;)

File: install.md (hunk 1 above)
Comment (rhatdan) [reply install.md]: Right I missed that

Comment (rhatdan) [reply install.md]: How about:

sudo curl -qsSL -o  /etc/cni/net.d/99-loopback.conf https://raw.githubusercontent.com/containers/libpod/master/cni/87-podman-bridge.conflist 

Then a simpleton like me, will not get fooled by the weird syntax.

Comment (nc-ruth) [reply install.md]: I'll make a new pull request for this.
I also noticed another issue with conman - apparently conman is not within $PATH when following the build by source.

Comment (nc-ruth) [reply install.md]: and I agree with you, for readability your solution is much better :)

**Pull request merged**